core/workspace_svg.js

```diff
@@ -715,11 +715,13 @@ Blockly.WorkspaceSvg.prototype.moveDrag = function(e) {
 };
 
 /**
- * Is the user currently dragging a block or scrolling the workspace?
+ * Is the user currently dragging a block or scrolling the flyout/workspace?
  * @return {boolean} True if currently dragging or scrolling.
  */
 Blockly.WorkspaceSvg.prototype.isDragging = function() {
   return Blockly.dragMode_ == Blockly.DRAG_FREE ||
+      (Blockly.Flyout.startFlyout_ &&
```

Comment:
The reason I use startFlyout_ here is because for flyout drags, the sub-workspace inside the flyout has isDragging() called. This sub-workspace doesn't have a link back to the flyout instance that owns it, just a boolean isFlyout member.

I think I could go through this.targetWorkspace.(toolbox_.)?flyout_ but it would be more code. Alternatively I could add a direct reference from the sub-workspace to the flyout instance.

What do you prefer?